GameTracker-Core/Controller.cs

```diff
@@ -0,0 +1,80 @@
+﻿using GameTracker_Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GameTracker_Core
+{
+    public class Controller
+    {
+        private Device device { get; set; }
+
+        public Controller()
+        {
+            device = new Device();
+            if (!System.IO.Directory.Exists("C:\\Users\\Pete\\AppData\\Roaming\\GameTrackerAgent"))
```

Comment:
Put the path to this directory in a `private static readonly` property.
Use `Path.Combine` to combine directory paths if needed.